docs/pages/core/wireguard-mesh.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Wireguard-Mesh
+layout: default
+nav_order: 3
+parent: PMD Core Components
+---
+
+Wireguard-Mesh
+===
+{: .no_toc }
+
+## Table of Contents
+{: .no_toc }
+
+- TOC
+{:toc}
+
+
+
+## Requirements
+* Public IPv4 address and a corresponding public DNS
+* Linux Kernel version 5.6 or wireguard kernel modules installed. (see
+  [https://www.wireguard.com/install/](https://www.wireguard.com/install/))
+* Docker (see [System Requirements](../index.md/#system-requirements))
+* docker-compose (see [System Requirements](../index.md/#system-requirements))
+
+### Request Certificate
+
+Please send your request for a PMD-S certificate via our contact form [https://material-digital.de/contact/](https://material-digital.de/contact/).
+Choose the subject "Network and security architecture".
```

Comment:
could use the direkt link: https://www.material-digital.de/contact/?subject=Netzwerk-+und+Sicherheitsarchitektur

Reply:
Schreib das gleich noch um.